services/core-api/app/api/services/email_service.py

```diff
@@ -0,0 +1,176 @@
+import requests, json
+
+from enum import Enum
+from flask import current_app
+
+from app.config import Config
+from app.api.constants import CORE_PURPLE_LOGO_BASE64_ENCODED
+
+
+class EmailBodyType(Enum):
+    HTML = 'html'
+    TEXT = 'text'
+
+
+class EmailEncoding(Enum):
+    BASE64 = 'base64'
+    BINARY = 'binary'
+    HEX = 'hex'
+    UTF8 = 'utf-8'
+
+
+class EmailPriority(Enum):
+    LOW = 'low'
+    NORMAL = 'normal'
+    HIGH = 'high'
+
+
+MDS_NO_REPLY_SIGNATURE = f'''
+<div>
+    <hr />
+    <p>This is a no-reply email address. If you need to contact the MDS team, please email us at: <a href="mailto: {Config.MDS_EMAIL}">{Config.MDS_EMAIL}</a>.</p>
+    <br />
+    <img src="{CORE_PURPLE_LOGO_BASE64_ENCODED}" width="320" height="106" alt="Core Logo">
+</div>
+'''
+
+
+# NOTE: Manage our Common Services application access here: https://getok.nrs.gov.bc.ca/app/apps/MDS
+# NOTE: See here for details: https://ches.nrs.gov.bc.ca/api/v1/docs
+class EmailService():
```

Comment:
There are just 3 methods implemented in this service, one to interface with each of the 3 CHES resources we need at the moment: authentication, health check, and send an email.